frontend/components/dashboard/DashboardValidatorManagementModal.vue

```diff
@@ -111,9 +110,8 @@ const onClose = () => {
 const mapIndexOrPubKey = (
   validators?: VDBManageValidatorsTableRow[],
 ): string[] => {
-  return uniq(
-    validators?.map(vali => vali.index?.toString() ?? vali.public_key) ?? [],
-  )
+  return Array.from(
+    new Set(validators?.map(vali => vali.index?.toString() ?? vali.public_key) ?? []))
```

Comment:
```suggestion
  return [...new Set(validators?.map(validator => validator.index?.toString() ?? validator.public_key)]
```
this would be another version, but plz take a look if this works too